app/helpers/application_helper.rb

```diff
@@ -20,29 +20,33 @@ def building_op_initial_state
     # Initial state here
     {
       buildings: ActiveModel::Serializer::CollectionSerializer.new(
-       current_building_operator.buildings, each_serializer: BuildingSerializer
-      ),
-      building_types: ActiveModel::Serializer::CollectionSerializer.new(
-       current_building_operator.building_types, each_serializer: BuildingTypeSerializer,
+       current_building_operator.buildings, each_serializer: BuildingSerializer,
        scope: {user_id: current_building_operator.id,
                user_type: 'BuildingOperator'}
       ),
+      questions: ActiveModel::Serializer::CollectionSerializer.new(
+        current_building_operator.questions, each_serializer: QuestionSerializer
+      ),
       userType: 'BuildingOperator'
     }
   end
 
   def rmi_user_initial_state
-    portfolios = Portfolio.all
     {
-      portfolios: portfolios,
+      portfolios: Portfolio.all,
       buildings: ActiveModel::Serializer::CollectionSerializer.new(
-        (portfolios.map { |p| p.buildings }).flatten, each_serializer: BuildingSerializer
+        Building.all, each_serializer: BuildingSerializer,
+        scope: {user_id: current_rmi_user.id,
+                user_type: 'RmiUser'}
       ),
       building_types: ActiveModel::Serializer::CollectionSerializer.new(
         BuildingType.all, each_serializer: BuildingTypeSerializer,
         scope: {user_id: current_rmi_user.id,
                 user_type: 'RmiUser'}
       ),
+      questions: ActiveModel::Serializer::CollectionSerializer.new(
```

Comment:
Are we running `QuestionSerializer` across a question redundantly? We may have addressed this last night, but I think this might be a different issue; it seems like `QuestionSerializer` is serializing all the applicable questions here, but also [in the `BuildingSerializer`](https://github.com/calblueprint/RMI/pull/150/files#diff-991b7bdb94f932280dceb6e40d491725R37). 

Is there a reason for this `questions` bit to exist, and if not, could we replace it with a function in our `LOAD_INITIAL_STATE` action that grabs all the questions from the `window.INITIAL_STATE.buildings` and places them in `store.questions`?